openapi/data_repository_service.swagger.yaml

```diff
@@ -192,97 +152,24 @@ definitions:
-        type: string
-        description: >-
-          An identifier unique to this Data Bundle.
-      name:
-        type: string
-        description: >-
-          A string that can be used to name a Data Bundle.
-      size:
-        type: string
-        format: int64
-        description: >-
-          The cumulative size, in bytes, of all Data Objects and Bundles listed in
-          the `contents` field.
-      created:
-        type: string
-        format: date-time
-        description: >-
-          Timestamp of Bundle creation in RFC3339.
-      updated:
-        type: string
-        format: date-time
-        description: >-
-          Timestamp of Bundle update in RFC3339, identical to create timestamp in
-          systems that do not support updates.
-      version:
-        type: string
-        description: >-
-          A string representing a version. (Some systems may use checksum, a
-          RFC3339 timestamp, or an incrementing version number.)
-      checksums:
-        type: array
-        description: |-
-          The checksum of the Data Bundle. At least one checksum must be provided.
-          
-          The Data Bundle checksum is computed over a sorted concatenation of all 
-          the checksums (names not included) within the top-level 'contents' of the
-          Bundle (not recursive). The list of Data Object or Bundle checksums are 
-          sorted alphabetically (hex-code) before concatenation and a further checksum
-          is performed on the concatenated checksum value.
-          Example below:
-          Data Ojects:
-            md5(DO1) = 72794b6d30bc86d92e40a1aa65c880b8
-            md5(DO2) = 5e089d29a18954e68a78ee6a3c6edabd
-          Data Bundle:
-          DB1 = md5( concat( sort( md5(DO1), md5(DO2) ) ) )
-              = md5( concat( sort( 72794b6d30bc86d92e40a1aa65c880b8, 5e089d29a18954e68a78ee6a3c6edabd ) ) )
-              = md5( concat( 5e089d29a18954e68a78ee6a3c6edabd, 72794b6d30bc86d92e40a1aa65c880b8 ) )
-              = md5( 5e089d29a18954e68a78ee6a3c6edabd72794b6d30bc86d92e40a1aa65c880b8 )
-              = f7a29a0422e7d870b10839ad6c985079
-        items:
-          $ref: '#/definitions/Checksum'
-      description:
-        type: string
-        description: A human readable description of the Data Bundle.
-      aliases:
-        type: array
-        description: >-
-          A list of strings that can be used to find other metadata 
-          about this Data Bundle from external metadata sources. These
-          aliases can be used to represent the Data Bundle's secondary
-          accession numbers or external GUIDs.
-        items:
-          type: string
-      contents:
-        type: array
-        description: >-
-          The list of Data Objects and Data Bundles contained by this Data Bundle.
-        items:
-          $ref: '#/definitions/BundleObject'
   Object:
     type: object
-    required: ['id', 'size', 'created', 'checksums', 'access_methods']
```

Comment:
@dglazer I forget the reason why `access_methods` was removed from `object`

Reply:
Because they're not required for bundles. (This is one of the small downsides of a single method. IMO, that's more than made up for by the upside of having DRS IDs be self-contained.)